kategory/src/test/kotlin/kategory/laws/TraverseLaws.kt

```diff
@@ -17,14 +17,14 @@ class TIF {
 }
 
 object TraverseLaws {
+    // FIXME(paco): cf is only required because AP::pure will crash the inliner
```

Comment:
Is this related to being a method reference lifted to a function and not an actual function?